virttest/utils_v2v.py

```diff
@@ -606,8 +606,23 @@ def get_driver_info(self, signed=True):
         cmd = "DRIVERQUERY"
         if signed:
             cmd += " /SI"
-        status, output = self.session.cmd_status_output(cmd)
-        return self.run_cmd(cmd)[1]
+        # Try 5 times to get driver info
+        output, count = '', 5
+        while count > 0:
+            logging.debug('%d times remaining for getting driver info' % count)
+            try:
+                output = self.session.cmd_output(cmd)
+            except Exception as detail:
+                logging.error(detail)
+                count -= 1
+                # Clean up output
+                self.session.cmd('cls')
+            else:
+                break
+        if not output:
+            logging.error('Fail to get driver info')
+        logging.debug("Command output:\n%s", output)
+        return output
 
```

Comment:
But utils_misc.wait_for  only deals with functions which needs more time to get result. In this situation, command could fail once or twice. During my test, the correct result most comes from the second run, sometimes the 3rd, therefore retrying several times might be necessary.